pkg/provider/bitbucketdatacenter/bitbucketdatacenter_test.go

```diff
@@ -406,11 +405,11 @@ func TestGetCommitInfo(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			ctx, _ := rtesting.SetupFakeContext(t)
-			bbclient, _, mux, tearDown, tURL := bbtest.SetupBBDataCenterClient(ctx)
+			bbclient, scmClient, mux, tearDown, tURL := bbtest.SetupBBDataCenterClient(ctx)
```

Comment:
```suggestion
			_, scmClient, mux, tearDown, tURL := bbtest.SetupBBDataCenterClient(ctx)
```

you can ignore old client, as its redundant here.